### PR TITLE
Add method to get mapping of available tick to frame index

### PR DESCRIPTION
**Pull request by chaosddp**
# Description

Add method to get mapping of available tick to frame index
## Linked issue(s)/Pull request(s)

- [issue_number](issue_link)

## Type of Change

- [ ] Non-breaking bug fix
- [ ] Breaking bug fix
- [x] New feature
- [x] Test
- [ ] Doc update
- [ ] Docker update

## Related Component

- [x] Simulation toolkit
- [ ] RL toolkit
- [ ] Distributed toolkit

## Has Been Tested

- OS:
  - [ ] Windows
  - [ ] Mac OS
  - [x] Linux
- Python version:
  - [x] 3.6
  - [ ] 3.7
- Key information snapshot(s):

## Needs Follow Up Actions

- [ ] New release package
- [ ] New docker image

## Checklist

- [x] Add/update the related comments
- [x] Add/update the related tests
- [ ] Add/update the related documentations
- [ ] Update the dependent downstream modules usage

### CODE REVIEW

File: tests/test_env.py [hunk 1]
```diff
@@ -293,6 +294,45 @@ def test_frame_index_to_ticks(self):
...
+            os.environ["DEFAULT_BACKEND_NAME"] = backend_name
+
+            max_tick = 10
+
+            env = Env(scenario="cim", topology="tests/data/cim/customized_config",
+                      start_tick=0, durations=max_tick)
+
+            run_to_end(env)
+
+            f2t_mapping = env.get_ticks_frame_index_mapping()
```
Comment (Jinyu-W) [inline tests/test_env.py:308]: t2f_mapping?

File: tests/test_env.py [hunk 2]
```diff
@@ -293,6 +294,45 @@ def test_frame_index_to_ticks(self):
...
+
+            # tick == frame index
+            self.assertListEqual([f for f in f2t_mapping.keys()], [f for f in range(max_tick)])
```
Comment (Jinyu-W) [inline tests/test_env.py:311]: f -> t for this statement?

File: tests/test_env.py (hunk 1 above)
Comment (chaosddp) [reply tests/test_env.py]: fixed

File: tests/test_env.py (hunk 2 above)
Comment (chaosddp) [reply tests/test_env.py]: fixed

**Comment by chaosddp** [comment]
Seems like ubuntu 16 docker images has been removed on github workflow, it keep show following message:

Can't find any online and idle self-hosted runner in the current repository, account/organization and enterprise that matches the required labels: 'ubuntu-16.04'
Waiting for a self-hosted runner to pickup this job...

**Pull request merged**